pages/guides/dwolla-js/instant-account-verification/configure-and-call-function.mdx

```diff
@@ -0,0 +1,131 @@
+| currentPage | `BankSearch` <br /> `BankLogin` <br /> `MFA` <br /> `SelectAccount` <br /> `SuccessIAV` <br /> `SelectVerificationMethod` <br /> `SubmitBankAccountDetails` <br /> `SuccessMicroDeposits`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| error       | **code** and **message**: <br /> `InvalidLogin` - "Please make sure your login or security information is correct." <br /> `AccountNotFound` - "Sorry, we’re unable to find your {BANK NAME} account. Please try again or use a different account." <br /> `UnsupportedSite` - "Sorry, that financial institution is not supported. If possible, please choose a different one or an alternative method for connecting your financial institution." <br /> `AlreadyLoggedIn` - "If you're currently logged in to your {BANK NAME} account, please log out and try again." <br /> `VisitSite` - "We’re unable to process your information because the {BANK NAME} site is currently requiring additional action from you. Please resolve this, then try again." <br /> `UnavailableSite` - "Sorry, there seem to be some technical difficulties while attempting to process your information. Please try again later." |
+
+##### **Example subscriber callback**
+
+```json
+{
+  "currentPage": "BankLogin",
+  "error": {
+    "code": "InvalidLogin",
+    "message": "Please make sure your login or security information is correct."
+  }
+}
+```
+
+#### `stylesheets`
+
+Dwolla provides a list of CSS classes available for styling certain elements of the IAV flow. These elements can be easily customized to match the look and feel of your application, and are included within the `options` JavaScript object of the function dwolla.iav.start(). You can specify one or many stylesheets as a list within the `stylesheets` attribute. By default, the elements within your specified container are responsive to any change in screen size.
+
+<a
+  href="https://www.dwolla.com/dwollajs-bank-verification"
+  target="_blank"
+  class="btn secondary medium"
```

Comment:
This link is being styled incorrectly. We could either display it as a normal link by removing the classes here or display it as a button by importing the Button component and using that